icevision/metrics/segmentation_accuracy/segmentation_accuracy.py

```diff
@@ -0,0 +1,70 @@
+__all__ = ["SegmentationAccuracy"]
+
+from icevision.imports import *
+from icevision.utils import *
+from icevision.data import *
+from icevision.metrics.metric import *
+
+
+class SegmentationAccuracy(Metric):
+    """A simple segmentation accuracy metric
+
+    Calculates segmentation accuracy.
+
+    # Arguments
+        ignore_class: class to ignore during metric computation (e.g. void class).
+
+    """
+
+    _seg_masks_gt = None
+    _seg_masks_pred = None
+
+    def __init__(self, ignore_class=None):
```

Comment:
Why don't we set this to `Void` as default?

Reply:
This is an arbitrary choice on my part - I thought the expected default behaviour would be not to ignore any class, though I can see why `Void` would make sense here. Happy either way.

Reply:
@potipot @fstroth @ai-fast-track wdyt?

Reply:
I'm not familiar with and never used `Void`. In context like this I also usually set var to None

Reply:
@lgvaz what are your 2 cents here?

Reply:
On another note though, why is this set as class attribute rather than instance, set on init?

Reply:
Does it matter?

Reply:
My intuition was right, take a look at this example:
![image](https://user-images.githubusercontent.com/52150545/155285682-0b63a56c-91be-4209-b3e2-8a1f16d49743.png)

Reply:
Wait, Ia m referring to `ignore_class` which is set as an instance attribute, not a class one.

Reply:
Oh, sorry I totally misunderstood the issue then.

Reply:
Np man.
When I asked
> Why don't we set this to Void as default?

I meant 
"Why don't we set this ignore_class to Void as default?"

Reply:
Thanks for the discussion guys.
Thinking about it more thoroughly, I am starting to convince myself that it could be a good idea to set `ignore_class=None` as it's done now. I am not sure but I think that `Void` is CamVid specific.